src/components/CalendarCell/CalendarCell.st.css

```diff
@@ -0,0 +1,152 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback, font, applyOpacity;
+}
+
+:import {
+    -st-from: "../Text/Text.st.css";
+    -st-default: TPAText;
+}
+
+/*Defaults*/
+:vars {
+    DefaultAlignment: left;
+    DefaultBorderWidth: 1px;
+    DefaultBorderOpacity: 1;
+    DefaultFontColor: color-5;
+    DefaultBackgroundColor: color-1;
+    DefaultBorderColor: color-2;
+    DefaultTodayColor: color-8;
+    DefaultFontSize: 16px;
+    DefaultFontStyle: "{theme: 'Body-M', size:'16px' lineHeight: '1em'}";
+}
+
+/*Overrides*/
+:vars {
+    Alignment: --overridable
+    BorderWidth: --overridable
+    BorderOpacity: --overridable
+    FontColor: --overridable
+    BackgroundColor: --overridable
+    BorderColor: --overridable
+    TodayColor: --overridable
+    FontStyle: --overridable
+    FontSize: --overridable
+}
+
+:vars {
+    CurrentAlignment: fallback(value(Alignment), value(DefaultAlignment));
+    CurrentBorderWidth: fallback(value(BorderWidth), value(DefaultBorderWidth));
+    CurrentBorderOpacity: fallback(value(BorderOpacity), value(DefaultBorderOpacity))
+    CurrentFontColor: color(fallback(value(FontColor), value(DefaultFontColor)));
+    CurrentBackgroundColor: color(fallback(value(BackgroundColor), value(DefaultBackgroundColor)));
+    CurrentBorderColor: color(fallback(value(BorderColor), value(DefaultBorderColor)));
+    CurrentTodayColor: color(fallback(value(TodayColor), value(DefaultTodayColor)));
+    CurrentFontStyle: fallback(value(FontStyle), font(value(DefaultFontStyle)));
+    CurrentFontSize: fallback(value(FontSize), value(DefaultFontSize));
+}
+
+.root {
+    -st-states: stretchable, timeType(enum(previousMonth, previousDays, currentDay, nextMonth))
+    border: value(CurrentBorderWidth) solid applyOpacity(value(CurrentBorderColor), value(CurrentBorderOpacity));
+    background-color: value(CurrentBackgroundColor);
+    text-align: value(CurrentAlignment);
+    width: 100%;
+    padding-top: 100%; /* 1:1 Aspect Ratio */
+    position: relative;
+}
+
+.innerContainer {
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    padding: 12px;
+}
+
+/* .childrenContainer {
+    margin-top: 10px;
+    display: flex;
+    flex-direction: column;
+    height: 67%;
+} */
```

Comment:
Is this needed?

Reply:
i also extract the children from the div container it was in.